### PR TITLE
feat: add fastcdr v2.1.3

### PR DESCRIPTION
**Pull request by wep21**
- add [fastcdr v2.1.3](https://github.com/eProsima/Fast-CDR/releases/tag/v2.1.3)

### CODE REVIEW

**Comment by wep21** [comment]
This requires the skip-url-stability-check label

File: modules/fastcdr/metadata.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "homepage": "https://github.com/eProsima/Fast-CDR",
+    "maintainers": [],
```
Comment (fmeum) [inline modules/fastcdr/metadata.json:3]: Would you be willing to add yourself as a maintainer for this BCR entry?

File: modules/fastcdr/2.1.3/patches/add_cmake_configure_file.patch [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
++        "src": attr.label(
++            allow_single_file = True,
++            mandatory = True,
++        ),
++        "out": attr.output(mandatory = True),
++        "defines": attr.string_list(),
++        "undefines": attr.string_list(),
++        "cmakelists": attr.label_list(allow_files = True),
++        "cmake_configure_file_py": attr.label(
++            cfg = "host",
```
Comment (fmeum) [inline modules/fastcdr/2.1.3/patches/add_cmake_configure_file.patch:59]: This configuration is deprecated and should be replaced with `exec`.

File: modules/fastcdr/2.1.3/MODULE.bazel [hunk 3]
```diff
@@ -0,0 +1,18 @@
+module(
+    name = "fastcdr",
+    version = "2.1.3",
+    compatibility_level = 1,
+)
+
+bazel_dep(
```
Comment (fmeum) [inline modules/fastcdr/2.1.3/MODULE.bazel:7]: Since Python and C++ rules are still shipped with Bazel, these two deps aren't needed.

Comment (wep21) [reply modules/fastcdr/2.1.3/MODULE.bazel:7]: remove unnecessary deps at https://github.com/bazelbuild/bazel-central-registry/pull/1345/commits/5f9045114168aa8ef27be366ae08977e562bf1bf

File: modules/fastcdr/2.1.3/patches/add_cmake_configure_file.patch (hunk 2 above)
Comment (wep21) [reply modules/fastcdr/2.1.3/patches/add_cmake_configure_file.patch]: change `host` to `exec` at https://github.com/bazelbuild/bazel-central-registry/pull/1345/commits/e6ff09dbc45e3c63f2ffd1df3b0f60cc3af130c3

File: modules/fastcdr/metadata.json (hunk 1 above)
Comment (wep21) [reply modules/fastcdr/metadata.json]: add myself as a maintainer at https://github.com/bazelbuild/bazel-central-registry/pull/1345/commits/3ae2520d4ec26e6ff4059a021dd699177cf0e1c4

**Review by fmeum — APPROVED** [review]

**Pull request merged**